introduction.md

```diff
@@ -0,0 +1 @@
+Hi, I'm Luis Llave, web developer. My favorite path in Code School is Python.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.